starky/src/starkinfo_codegen.rs

```diff
@@ -416,6 +416,43 @@ fn find_muladd(exp: &Expression) -> Expression {
     exp.clone()
 }
 
+/// Iteratively traverse a tree in postorder, so that the process stack doesn't overflow.
```

Comment:
is there any sample code that will raise the stack overflow? hence we can add that as an unit test.